api/src/main/java/org/openmrs/module/insuranceclaims/api/dao/InsurancePolicyDao.java

```diff
@@ -0,0 +1,12 @@
+package org.openmrs.module.insuranceclaims.api.dao;
+
+import org.openmrs.module.insuranceclaims.api.model.InsurancePolicy;
+
+public interface InsurancePolicyDao {
+
+	InsurancePolicy getInsurancePolicyById(Integer id);
```

Comment:
move it to CRUD interface 'T extends BaseOpenmrsData>

Reply:
Thanks. I will extend the OpenmrsDataDAO interface.